### PR TITLE
Implement eclipselink extension with Quarkus

### PR DESCRIPTION
**Pull request by jbonofre**
This is a simple implementation of Polaris Eclipselink extension with Quarkus.

### CODE REVIEW

File: extension/persistence/eclipselink-quarkus/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ */
+
+plugins {
+  alias(libs.plugins.quarkus)
+  id("polaris-server")
+}
+
+dependencies {
+  implementation(project(":polaris-core"))
+  // TODO @RuntimeCandidate is in polaris-service-quarkus, maybe move a polaris-spi module ?
```
Comment (dimas-b) [inline extension/persistence/eclipselink-quarkus/build.gradle.kts:27]: Yeah, it would be nice to factor SPI out of `polaris-core` too

Comment (jbonofre) [reply extension/persistence/eclipselink-quarkus/build.gradle.kts:27]: Yes, agree. 
I will do that after the rebase.

File: extension/persistence/eclipselink-quarkus/src/main/java/org/apache/polaris/extension/persistence/impl/eclipselink/EclipseLinkPolarisMetaStoreManagerFactory.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+/**
+ * The implementation of Configuration interface for configuring the {@link PolarisMetaStoreManager}
+ * using an EclipseLink based meta store to store and retrieve all Polaris metadata. It can be
+ * configured through persistence.xml to use supported RDBMS as the meta store.
+ */
+@ApplicationScoped
+@RuntimeCandidate
+@LookupIfProperty(name = "polaris.persistence.metastore-manager.type", stringValue = "eclipselink")
```
Comment (adutra) [inline extension/persistence/eclipselink-quarkus/src/main/java/org/apache/polaris/extension/persistence/impl/eclipselink/EclipseLinkPolarisMetaStoreManagerFactory.java:38]: Sorry I'm late to this PR. Right now the value for selecting this metastore is `eclipse-link` with a dash.

Comment (jbonofre) [reply extension/persistence/eclipselink-quarkus/src/main/java/org/apache/polaris/extension/persistence/impl/eclipselink/EclipseLinkPolarisMetaStoreManagerFactory.java:38]: Thanks, I'm fixing directly on the `QUARKUS` branch.

**Pull request merged**